.github/workflows/CI.yml

```diff
@@ -35,14 +36,18 @@ jobs:
         rustflags:
 
     - name: Install Nextest test runner
-      uses: taiki-e/install-action@nextest
+      run: cargo install cargo-nextest
```

Comment:
Why did you remove the previous action? The one you've replaced it with will execute at every CI pipeline run, which is not what we want.

Reply:
Okay i see .Let me fix that